packages/firebase_app_check/firebase_app_check_platform_interface/lib/src/platform_interface/platform_interface_firebase_app_check.dart

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:firebase_core/firebase_core.dart';
+import 'package:meta/meta.dart';
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
+import '../method_channel/method_channel_firebase_app_check.dart';
+
+abstract class FirebaseAppCheckPlatform extends PlatformInterface {
+  /// The [FirebaseApp] this instance was initialized with.
+  FirebaseAppCheckPlatform({this.appInstance}) : super(token: _token);
+
+  /// Create an instance using [app] using the existing implementation
+  factory FirebaseAppCheckPlatform.instanceFor({required FirebaseApp app}) {
+    // Only the default app is supported on App Check.
```

Comment:
is that true? ios documentation at least implies secondary apps are supported https://firebase.google.com/docs/reference/ios/firebaseappcheck/api/reference/Classes

same on android https://firebase.google.com/docs/reference/android/com/google/firebase/appcheck/FirebaseAppCheck#public-method-summary

Reply:
With the current API there's some limitations in supporting this cross platform (some major underlying SDK differences making things difficult) so for the initial implementation of App Check in FlutterFire it won't be supported.